src/components/featured-place-card/featured-place-card-styles.module.scss

```diff
@@ -0,0 +1,162 @@
+@import 'styles/settings';
+@import 'styles/typography-extends';
+@import 'styles/ui.module';
+
+$card-content-max-height: 220px;
+
+.container {
+  pointer-events: none;
+  position: absolute;
+  bottom: 0;
+  display: flex;
+  width: 100%;
+  min-height: 380px;
+  background-color: transparent;
+  overflow: hidden;
+}
+
+.content {
+  position: relative;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 100%;
+}
+
+.navigation {
+  width: 100%;
+  height: 45px;
+  max-width: $card-max-width;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  pointer-events: all;
+
+  .backToMap {
+    display: flex;
+    flex-direction: row;
+    cursor: pointer;
+  
+    .text {
+      @extend %title;
+      color: $white;
+      margin: 0;
+    }
+  
+    .arrowIcon {
+      top: 2px;
+      position: relative;
+      transform: rotate(180deg);
+      margin-right: 30px;
+    }
+  }
+
+  .placesNavigator {
+    width: 100px;
+    padding: 0 10px;
+    height: 100%;
+    background-color: $white;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+
+    .arrowWrapper {
```

Comment:
I don't think we need to nest it; we're not overriding it

Reply:
Not sure what you mean @aabdaab can you clarify this comment?

Reply:
so, I meant that we don't have to nest `arrowWrapper` class inside `placesNavigator` (as we are not overriding this class)

Reply:
Ah ok. Well is just a matter of taste, I mean if you look closer the whole `scss` file I use to nest the styles to mimic the `html` structure, to have a grasp of the node tree structure when writing the styles.
For example this structure on the styles
```scss
.navigation {
  .backToMap {}
  .placesNavigation {}
}
```
resembles this one from the markup
```html
<nav className={styles.navigation}>
  <div className={styles.backToMap}/>
  <div className={styles.placesNavigation}/>
</nav>
```
Does it makes sense?

Reply:
I'm not sure about that approach. Generally, nesting css classes means that you're overriding something, i.e. my first thought when I see 
```
.navigation {
  .backToMap {}
}
```
is 'ok, so where is the original `backToMap` class and what are you overriding?'
Also, have a look at this [article](http://thesassway.com/intermediate/avoid-nested-selectors-for-more-modular-css), it talks about a more modular css (in our case it is already modular as we've got css-modules but it's more of a matter of convention at this point)

Reply:
Well when I see nested classes I think on namespacing :)
But yeah probably what we are talking about here is about setting some sort of `best practice` for the project. As we are already being modular and namespacing by default with `css-modules` we are avoiding a lot of the problems that will arise in other context. But we might want to define some convention for the styles.
Do you think having some conversation about this would be beneficial? @aabdaab , @kfilip94 ?

Reply:
Well, we don't need to nest things there, but I also understand that it can help to reflect the HTML structure. I don't have a strong opinion about one or another approach, I used both sometimes. If you want we can pick the one approach but to me, it is not something super important. Just use the approach that you prefer. :wink:

Reply:
Well, unless you want to nest more than two levels what is redundant for sure and can make the code less readable.

Reply:
Let's have a meeting next week -- it's not that something is breaking because of this